impl/src/main/resources/org/pentaho/commons/metadata/mqleditor/editor/xul/mainFrame.xul

```diff
@@ -124,16 +124,16 @@
         </tree>
 
 
-        <hbox id="conditionsButtonContainer" width="575">
+        <toolbar id="conditionsButtonContainer">
           <label value="${conditions}" flex="1" />
-          <spacer id="conditionsSpacer" flex="1" />
-          <button id="conditionDown" disabled="true"
+          <toolbarspacer id="conditionsSpacer" flex="1" />
+          <toolbarbutton id="conditionDown" disabled="true"
```

Comment:
Are all the attributes supported by `button` also supported by `toolbarbutton`?

Reply:
https://github.com/pentaho/pentaho-platform-plugin-dashboards/pull/570/files#r1341710831

Reply:
I take your answer to mean: "As answered elsewhere, no, not all attributes are supported. However, the only unsupported attribute, `pen:disabledimage`, is not used in these instances.".

Reply:
Yes, I should have phrased it like that.